sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/compiler/PhaseOne.scala

```diff
@@ -11,13 +12,22 @@ import com.digitalasset.daml.lf.language.{LookupError, PackageInterface}
 import com.digitalasset.daml.lf.speedy.Compiler.{ProfilingMode, StackTraceMode, CompilationError}
 import com.digitalasset.daml.lf.speedy.SBuiltinFun._
 import com.digitalasset.daml.lf.speedy.SValue._
-import com.digitalasset.daml.lf.speedy.SExpr0._
+import com.digitalasset.daml.lf.speedy.compiler.SExpr0._
 import com.digitalasset.daml.lf.speedy.{SExpr => t}
 import com.daml.nameof.NameOf
 
 import scala.annotation.tailrec
 
-private[speedy] object PhaseOne {
+/** Initial Conversion (Phase of the speedy compiler pipeline)
+  *
+  * This compilation phase transforms from LF to SExpr.
```

Comment:
```suggestion
  * This compilation phase transforms from LF to SExpr0.
```

(If I'm reading the comment in SExpr0 correctly)